### ISSUE TITLE
Reference to &lcc-3166-1;NorthMacedonia

### ISSUE THREAD

**Issue by mereolog** [edmcouncil/fibo#1669]
fibo-fbc-fct-mkti;Exchange-XMAE (MACEDONIAN STOCK EXCHANGE) refers to &lcc-3166-1;NorthMacedonia - see below - which does not exist, well at least it is not to be found in the current lcc-3166-1.

I think it should refer to &lcc-3166-1;Macedonia

```
<owl:NamedIndividual rdf:about="&fibo-fbc-fct-mkti;Exchange-XMAE">
	<rdf:type rdf:resource="&fibo-fbc-fct-mkt;OperatingLevelMarket"/>
	<rdf:type rdf:resource="&fibo-fbc-pas-fpas;FinancialServiceProvider"/>
        <rdfs:label>MACEDONIAN STOCK EXCHANGE</rdfs:label>
        <fibo-fbc-fct-mkt:hasExchangeName>MACEDONIAN STOCK EXCHANGE</fibo-fbc-fct-mkt:hasExchangeName>
        <fibo-fbc-fct-mkt:operatesInCountry rdf:resource="&lcc-3166-1;NorthMacedonia"/>
        <fibo-fbc-fct-mkt:operatesInMunicipality rdf:resource="&fibo-fbc-fct-bci;Skopje"/>
        <fibo-fnd-plc-vrt:hasWebsite rdf:datatype="&xsd;anyURI">http://www.mse.org.mk</fibo-fnd-plc-vrt:hasWebsite>
</owl:NamedIndividual>
```

Perhaps https://spec.edmcouncil.org/fibo/ontology/BE/GovernmentEntities/EuropeanJurisdiction/SouthernEuropeGovernmentEntitiesAndJurisdictions/NorthMacedonianEntity should also be revised.

**Comment by ElisaKendall** [comment]
@mereolog Hi Pawel, the revision to LCC for an LCC 1.2 version, which does include North Macedonia, is what Pete used to generate the FpML update.  It was supposed to have been published before the holiday, thus before the Q4 release was complete.  I don't know what the hold-up is but will investigate.  That version includes the reference to the original Macedonia for backwards compatibility, so I don't believe it will cause issues for the individuals in BE, but will test with it.

**Comment by ElisaKendall** [comment]
@merelog OMG has published the -versioned- LCC 1.2 ontologies, but they are still working on the non-versioned ones.  Mariano (the OMG standards catalog publishing person) let me know this morning that it will be done today or tomorrow latest.  We should recheck it once that's done, but hopefully you'll be able to close this issue with no change as a consequence.

**Comment by ElisaKendall** [comment]
@merelog This has now been corrected and the latest LCC 1.2 ontologies are published, including the non-versioned IRIs.  The referenced FpML rate for North Macedonia should resolve properly now.  Having said this, now that Macedonia is deprecated in LCC, we should revise its usage in currency codes and southern European governments and jurisdictions.
